packages/expo-modules-core/expo-module.config.json

```diff
@@ -3,5 +3,8 @@
   "platforms": ["ios", "android"],
   "ios": {
     "podspecPath": "./ExpoModulesCore.podspec"
+  },
+  "android": {
+    "modules": ["expo.modules.contacts.ContactsModule"]
```

Comment:
This seems to be the wrong `expo-module.config.json` :smile:

Reply:
Good catch 😂 👀